dashboard/app/helpers/users_helper.rb

```diff
@@ -108,7 +110,7 @@ def summarize_user_progress_for_all_scripts(user)
       script_levels.each do |sl|
         sl.level_ids.each do |level_id|
           # if we have a contained level, use that to represent progress
-          contained_level_id = Level.find(level_id).contained_levels.try(:first).try(:id)
+          contained_level_id = Level.cache_find(level_id).contained_levels.try(:first).try(:id)
```

Comment:
Is there any linting we could easily/reasonably do to catch stuff like this? I hadn't even known about cache_find or that I should use it.